app/presenters/world_index_presenter.rb

```diff
@@ -1,9 +1,42 @@
 class WorldIndexPresenter
+  include ActionView::Helpers::UrlHelper
+
+  delegate :count, to: :world_locations, prefix: true
+
   def initialize(world_index)
     @world_index = world_index
   end
 
   def title
     @world_index.content_item.title
   end
+
+  def grouped_world_locations
+    world_locations
+      .group_by { |location| location_group(location) }
+      .sort
+  end
+
+  def world_location_link(world_location)
```

Comment:
Not sure what to do about this, I quite like it as is because it's easily testable. What do you mean about the tracking, and how strongly do you feel about it?

Reply:
We can refactor it if/when tracking is required. Fine to leave for now 👍